src/ui/actions_main_area.ui

```diff
@@ -224,125 +299,205 @@
+           <property name="text">
+            <string>Command (down):</string>
+           </property>
+          </widget>
+         </item>
+         <item row="3" column="1" colspan="5">
+          <widget class="QLineEdit" name="lineEdit_action_command_down">
+           <property name="enabled">
             <bool>true</bool>
            </property>
-           <property name="checked">
-            <bool>false</bool>
+           <property name="minimumSize">
+            <size>
+             <width>0</width>
+             <height>0</height>
+            </size>
+           </property>
+           <property name="toolTip">
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a plain text command to be send directly to the MUD server when this button is clicked.&lt;/p&gt;&lt;p&gt;A menu does not act in the same way as the action of clicking on it is normally to display the contents of the menu so this will not show should a menu (a button &lt;i&gt;Group&lt;/i&gt; item be selected).&lt;/p&gt;&lt;p&gt;If this is a button that has been set as a &lt;i&gt;push-down button&lt;/i&gt; then what is entered here is sent on the &lt;b&gt;first&lt;/b&gt; press of the button, i.e. going from the &lt;b&gt;off to on&lt;/b&gt; state - in this case a second entry box is enabled and what is entered in &lt;i&gt;that one&lt;/i&gt; is send when the button is clicked a second time to go from the &lt;b&gt;on to off&lt;/b&gt; state.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+           </property>
+          </widget>
+         </item>
+         <item row="4" column="0">
+          <widget class="QLabel" name="label_action_command_up">
+           <property name="text">
+            <string>Command (up):</string>
            </property>
-           <layout class="QHBoxLayout" name="horizontalLayout_4">
-            <property name="spacing">
-             <number>0</number>
-            </property>
-            <property name="margin">
-             <number>0</number>
-            </property>
-           </layout>
           </widget>
          </item>
+         <item row="4" column="1" colspan="5">
+          <widget class="QLineEdit" name="lineEdit_action_command_up">
+           <property name="enabled">
+            <bool>true</bool>
+           </property>
+           <property name="minimumSize">
+            <size>
+             <width>0</width>
+             <height>0</height>
+            </size>
+           </property>
+           <property name="toolTip">
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this is a button that has been set as a &lt;i&gt;push-down button&lt;/i&gt; then what is entered here is sent on the &lt;b&gt;second&lt;/b&gt; press of the button, i.e. going from the &lt;b&gt;on to off&lt;/b&gt; state - in this case the other entry box is only sent every other click as the button goes from the &lt;b&gt;off to on&lt;/b&gt; state.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
``have to RTFM in order to use something`` and ``interface be super intuitive`` are mutually exclusive, and I think we should focus on the latter, not the former :wink: . The reason you were having difficulty _to begin_ with was because the interface was designed badly - that's it, Qt Designer failed at step # 1.